src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -84,7 +84,18 @@ internal virtual BoundExpression BindSwitchExpressionCore(SwitchExpressionSyntax
 
         private BoundPattern BindDiscardPattern(DiscardPatternSyntax node, TypeSymbol operandType, bool hasErrors, DiagnosticBag diagnostics)
         {
-            // PROTOTYPE(patterns2): give an error if there is a bindable `_` in scope.
+            // give an error if there is a bindable `_` in scope.
+            var lookupResult = LookupResult.GetInstance();
+            var name = node.UnderscoreToken.ValueText;
+            HashSet<DiagnosticInfo> useSiteDiagnostics = null;
+            this.LookupSymbolsWithFallback(lookupResult, name, arity: 0, ref useSiteDiagnostics, options: LookupOptions.AllMethodsOnArityZero);
```

Comment:
I think LookupSymbols alone would be suitable here. The doc comment for LookupSymbolsWithFallback indicates it goes through more effort in the case of failure to provide more information, but I don't think we care what happens in the case of failure.